app/controllers/api/v1/accounts/statuses_pins_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::StatusesPinsController < Api::BaseController
```

Comment:
There is already an API for getting an account's pinned toots... No need to add a new API.

Reply:
Thank you for reply! I will fix it.